authors/settings.py

```diff
@@ -40,10 +41,12 @@
     'corsheaders',
     'django_extensions',
     'rest_framework',
-
+    'taggit',
```

Comment:
If this is for tagging articles, remove it since you're not implementing that in this task